product-service/src/getProductsById/index.js

```diff
@@ -0,0 +1,28 @@
+import { getDbProductById } from '../utils/dbFunctions';
+import { withCorsHeaders } from '../utils/middleware';
+
+export const getProductsById = async (event) => {
+  const { id } = event?.pathParameters || {};
+
+  if (!id) {
+    return {
+      statusCode: 400,
+      message: 'Product id not specified',
```

Comment:
you maybe need to put this message (as well as the message for not found response) into `body` field, currently I see this, but I should probably see the message
<img width="1492" alt="image" src="https://github.com/SirGooseTheNaughty/aws-practice-be/assets/22725775/d1b171ae-02a5-4819-8690-392dc8cd17af">